deploy/olm-catalog/ibm-block-csi-operator/1.5.0/bundle-1.5.0.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.5,v4.6"
```

Comment:
Is it close we are not supporting v4.7 ?

Reply:
good question.
we can always add it in case we decide to add support for it

Reply:
okay

Reply:
if it will be added later we will need to updated here as well (not only in the publication files..)

Reply:
sounds good